packages/snowpack/src/config.ts

```diff
@@ -171,7 +173,7 @@ function expandCliFlags(flags: CLIFlags): DeepPartial<SnowpackConfig> {
       continue;
     }
     if (configSchema.properties.buildOptions.properties[flag]) {
-      result.devOptions[flag] = val;
+      result.buildOptions[flag] = val;
```

Comment:
Bugfix! Now `--no-source-maps` and `--no-minify` work as CLI flags.

🤔 should we add all these flags to `snowpack --help`?